core/chains/evm/gas/block_history_estimator.go

```diff
@@ -187,7 +189,12 @@ func (b *BlockHistoryEstimator) GetLegacyGas(_ []byte, gasLimit uint32, maxGasPr
 		return nil, 0, errors.New("BlockHistoryEstimator is not started; cannot estimate gas")
 	}
 	if gasPrice == nil {
-		return nil, 0, errors.New("BlockHistoryEstimator has not finished the first gas estimation yet, likely because a failure on start")
+		if !b.getInitialFetch() {
+			return nil, 0, errors.New("BlockHistoryEstimator has not finished the first gas estimation yet, likely because a failure on start")
+		}
+		b.logger.Warnw(fmt.Sprintf("BlockHistoryEstimator couldn't provide an estimation. This is likely because" +
```

Comment:
I would also suggest logging the blocks numbers in history as structured data here.